examples/cognite_core/_api/cognite_cad_model.py

```diff
@@ -372,6 +381,7 @@ def histogram(
             | Sequence[str | tuple[str, str] | dm.NodeId | dm.DirectRelationReference]
             | None
         ) = None,
+        type_: Literal["CAD", "Image360", "PointCloud"] | list[Literal["CAD", "Image360", "PointCloud"]] | None = None,
```

Comment:
We need to make a type alias to avoid repeating the type. Too easy that someone updates one and forgets the other.

Reply:
Oh, this code is generated?

Reply:
That is correct, everything inside `examples` is generated.

Reply:
We state that in the description of the Pygen PRs going forward.